packages/tables/examples/paginated-table.md

```diff
@@ -36,34 +36,30 @@ const getPagedData = (data, currentPage, pageSize) => {
 
 <div>
   <Table size="small">
-    <Caption>
-      <XL tag={Caption} style={{ marginBottom: DEFAULT_THEME.space.sm }}>
-        Paginated Tickets
-      </XL>
+    <Caption style={{ marginBottom: DEFAULT_THEME.space.sm }}>
```

Comment:
I noticed that in production, the margin bottom is applied to the `<XL>` immediately below this line. What do you think about removing the margin here, and applying it back to the `<XL>`?

Before (production): 
<img width="1162" alt="Screen Shot 2019-12-06 at 2 36 42 PM" src="https://user-images.githubusercontent.com/1811365/70361423-6630db80-1836-11ea-9a3b-c4f58f983c5c.png">

Current (feature branch), the spacing between the `<Caption>` and `<XL>` is very close:
<img width="1122" alt="Screen Shot 2019-12-06 at 2 40 34 PM" src="https://user-images.githubusercontent.com/1811365/70361424-6630db80-1836-11ea-8484-88f3994b2e0e.png">